### PR TITLE
[ACS-10076] [ACA] Search input clears when using special search with text:value

### PR DESCRIPTION
**Pull request by dominikiwanekhyland**
**JIRA ticket link or changeset's description**
https://hyland.atlassian.net/browse/ACS-10076

### CODE REVIEW

File: projects/aca-content/src/lib/utils/aca-search-utils.ts [hunk 1]
```diff
@@ -117,7 +117,7 @@ export function extractSearchedWordFromEncodedQuery(encodedQuery: string): strin
       ? userQuery
           .split('AND')
           .map((searchCondition) => {
-            const searchTerm = searchCondition.split('"')[1];
+            const searchTerm = searchCondition.includes('"') ? searchCondition.split('"')[1] : searchCondition.trim();
```
Comment (AleksanderSklorz) [inline projects/aca-content/src/lib/utils/aca-search-utils.ts:120]: please add unit tests to test it

Comment (DenysVuika) [reply projects/aca-content/src/lib/utils/aca-search-utils.ts:120]: I think that needs some sort of regex here, for instance

```
const userQuery = `title:"Hello World" AND status:active AND "complex phrase"`;

const searchTerms = userQuery
  .split(/\s+AND\s+/i) // split safely, case-insensitive, allow spaces
  .map((condition) => {
    const match = condition.match(/"([^"]+)"|(\S+)/); 
    return match ? match[1] ?? match[2] : '';
  })
  .filter(Boolean); // remove empty strings

console.log(searchTerms);
// ["Hello World", "status:active", "complex phrase"]
```

**Review by AleksanderSklorz — APPROVED** [review]

File: projects/aca-content/src/lib/utils/aca-search-utils.ts (hunk 1 above)
Comment (dominikiwanekhyland) [reply projects/aca-content/src/lib/utils/aca-search-utils.ts:120]: @DenysVuika I tried a full regex solution, but it quickly became very complex due to all the edge cases in the unit tests — quoted vs unquoted terms, optional field prefixes, multiple AND conditions, trailing asterisks, etc. Handling everything correctly would require many additional if statements and special cases anyway, so I think keeping the current approach with minimal regex for parsing quoted vs unquoted terms is safer, easier to read, and more maintainable.
